### DISCUSSION TITLE
Multiple languages series

### DISCUSSION THREAD

**Discussion by donotos** [gotson/komga#1775] [Ideas]
Hello.
I got some mangas in two languages and i'd like to have them in the same serie like mangadex do for example.
it could be donne with patterneds subfolders (en, ja, es, fr...) like the oneshot fubfolder and adding an option in the serie page to filter books by language.
Thanks

**Comment by gotson** [comment]
Just create 2 series. They are not the same series if they are not the same language, you would have some trouble sorting the volumes/chapter anyway.
